### PR TITLE
Simplify build

### PR DESCRIPTION
**Pull request by jsamsa**
This change was to move functionality found in build.sh files and Grunt configuration into the npm build with the goal of simplifying the build process by reducing the number of tools required.

### CODE REVIEW

**Comment by vertein** [comment]
I like it.  Does anything change in our documentation here: https://github.com/UW-Madison-DoIT/uw-frame/blob/master/README.md or  https://github.com/UW-Madison-DoIT/uw-frame/blob/master/docs/markdown/quickstart.md which results in the io page here being updated: http://uw-madison-doit.github.io/uw-frame/latest/#/md/quickstart
?

**Comment by jsamsa** [comment]
- Fixed missing rimraf issue
- Updated documentation
- Modified Dockerfile

File: Dockerfile [hunk 1]
```diff
@@ -1,17 +1,19 @@
 FROM node:5.7.1-slim
 MAINTAINER Tim Levett <tim.levett@wisc.edu>
-RUN npm install -g grunt-cli superstatic
+RUN npm install -g superstatic
 
 RUN apt-get update && apt-get install -y bzip2 && apt-get install -y git
 
 # Add frame
+COPY tools /build/tools
+copy docs /build/docs
```
Comment (paulerickson) [inline Dockerfile]: uppercase `COPY` is more [conventional](https://docs.docker.com/engine/reference/builder/#/format)

**Comment by paulerickson** [comment]
:+1: less is more!

**Pull request merged**